pkg/install/install.go

```diff
@@ -100,12 +101,36 @@ func DefaultImage() string {
 	return image
 }
 
+func validMailAddress(address string) (string, bool) {
+	addr, err := mail.ParseAddress(address)
+	if err != nil {
+		return "", false
+	}
+	return addr.Address, true
+}
+
 func Install(ctx context.Context, image string, opts *Options) error {
 	// I don't want these errors on the screen. Probably a better way to do this.
 	klog.SetOutput(io.Discard)
 	klogv2.SetOutput(io.Discard)
 	utilruntime.ErrorHandlers = nil
 
+	if opts.Config.LetsEncrypt != nil && *opts.Config.LetsEncrypt == "production" {
+		if opts.Config.LetsEncryptEmail == "" {
+			result, err := pterm.DefaultInteractiveTextInput.WithMultiLine(false).Show("Enter your email address for Let's Encrypt (required for production)")
```

Comment:
Should this be replaced by something similar to https://github.com/acorn-io/acorn/blob/main/pkg/cli/credential_login.go#L54-L70 ?

Reply:
What you did here looks fine when I run it locally. I don't see a need to to switch to survey